lib/cli/handlers/status.js

```diff
@@ -0,0 +1,51 @@
+const path = require('path');
+const chalk = require('chalk');
+
+const { logRed, logGreen, getComposeFileOrFlagsHandler } = require('./../methods');
+const { Polygon } = require('../../services/polygon');
+const { Terraform } = require('./../../../lib/services/terraform');
+const { prepareArgsForPolygonOps } = require('./common');
+
+async function status({ fileMode = false, __fileDirname = '', awsProfile, sshPublicKey: _sshPublicKey,
+    orbsAddress, orbsPrivateKey, region, nodeSize, nodeCount, publicIp,
```

Comment:
It seems like we don't need some of these parameters.

Reply:
We talked about scheduling a cleanup of parameters, We will let @ronnno prioritize work on that clean up as well on getting the tests slimmed down and better represent our use one use case for the public network. ( I believe the private network test with 3-4 nodes is not neccesary anymore?) Let's setup some small discussin about what cleanups we want to perform over here and which variables should be removed as well from the `node.json`